includes/class-wc-payments-customer-service.php

```diff
@@ -128,25 +125,18 @@ public function create_customer_for_user( $user, $name, $email ) {
 	 *
 	 * @param string  $customer_id WCPay customer ID.
 	 * @param WP_User $user        WordPress user.
-	 * @param string  $name        Customer's full name.
-	 * @param string  $email       Customer's email address.
+	 * @param array   $customer_data Customer data.
 	 *
 	 * @return string The updated customer's ID. Can be different to the ID parameter if the customer was re-created.
 	 *
 	 * @throws API_Exception Error updating the customer.
 	 */
-	public function update_customer_for_user( $customer_id, $user, $name, $email ) {
-		$description = $this->build_description_string( $user, $name );
-
+	public function update_customer_for_user( $customer_id, $user, $customer_data ) {
```

Comment:
Same here: parameter type hints + return type hint.

Reply:
Thanks!

I agree - normally I request the type hints for methods when we are changing the signature. Making it globally not an option at the moment - we need to reach some critical mass of type-hinted interfaces.